cmd/flags.go

```diff
@@ -48,18 +49,22 @@ func getCommandList(cmd *cobra.Command) (commandList []string, err error) {
 	return commandList, nil
 }
 
-func getRegionList(cmd *cobra.Command) (regionList []string, err error) {
-	if regionList, err = cmdutil.GetFlagStringSlice(cmd, "region"); err != nil {
-		return nil, err
+func getRegionList(cmd *cobra.Command) ([]string, error) {
+	regions, err := cmdutil.GetFlagStringSlice(cmd, "region")
+	if err != nil {
+		return []string{}, err
 	}
 
-	if len(regionList) == 0 { // If no region is specified, attempt to look it up
-		if env, exists := os.LookupEnv("AWS_REGION"); exists {
-			return []string{env}, nil
-		}
+	if len(regions) != 0 {
+		return regions, nil
+	}
+
+	defaultRegion, exists := os.LookupEnv("AWS_REGION")
```

Comment:
I know **AWS_REGION** used to be used in the was SDK but I _think_ [**AWS_DEFAULT_REGION** is what's used now](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html#cli-configure-quickstart-region). I read that **AWS_REGION** is still used for some sdks/libraries and that might be a legacy thing, not sure. Maybe we should support both?

Reply:
[Looks like you're correct.](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-envvars.html#envvars-list) Good catch, we can probably just move to AWS_DEFAULT_REGION.

Reply:
Yeah it looks like even the AWS CLI has fully moved to AWS_DEFAULT_REGION. I will change this to use that instead.

Reply:
Ugh. I accidentally pasted the wrong link. Thanks, @sendqueery

Reply:
AWS_DEFAULT_REGION is the officially supported setting but AWS_REGION is also used in lots of community tools and takes precedence in the cli.

```
# I set up instances in 3 different regions
# my profile is configured for us-west-2
export AWS_DEFAULT_REGION=us-west-1
export AWS_REGION=us-east-1

aws ec2 describe-instances --query 'Reservations[].Instances[].Placement.AvailabilityZone' --output text
us-east-1c
unset AWS_REGION
aws ec2 describe-instances --query 'Reservations[].Instances[].Placement.AvailabilityZone' --output text
us-west-1c
unset AWS_DEFAULT_REGION
aws ec2 describe-instances --query 'Reservations[].Instances[].Placement.AvailabilityZone' --output text
us-west-2b
```

Reply:
Will make sure we support both env variables then

Reply:
not to beat a dead horse but AWS_REGION still seems legit I guess: https://github.com/aws/aws-sdk-go/#configuring-aws-region